docs/modules/usage/how-to/gke-example.md

```diff
@@ -0,0 +1,77 @@
+apiVersion: apps/v1
```

Comment:
we already have a Kubernetes doc here (it's called openshift-example.md, we could maybe rename it).

I'd suggest putting this in the same doc, maybe with a GKE heading

Reply:
I've added the required changes.

Reply:
You can delete this file now